meta-luneos/recipes-webos-ose/webos-connman-adapter/webos-connman-adapter/webos-connman-adapter.sh

```diff
@@ -17,11 +17,6 @@
 
 # To provide enough time for inbuilt adapter to be configured
 
-# first, if the cdc_ether isn't built as a module, there's nothing to do
-if ! zgrep -qs CONFIG_USB_NET_CDCETHER=m /proc/config.gz; then
-    exit 0
-fi
-
```

Comment:
This is a fix we introduced earlier: https://github.com/webOS-ports/meta-webos-ports/commit/9c7f483dcd7cc12d5d777b822e42afdb30003ac4 (sorry I didn't put any explanation here :( ). This early exit avoids a failure loop when trying to modprobe cdc_ether.

Reply:
OK will add it back